target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -224,7 +225,8 @@ ramips_setup_macs()
 		wan_mac=$(macaddr_add "$(cat /sys/class/net/eth0/address)" 2)
 		;;
 	wevo,11acnas|\
-	wevo,w2914ns-v2)
+	wevo,w2914ns-v2|\
+	zio,freezio)
 		wan_mac=$(mtd_get_mac_binary factory 0x2e)
```

Comment:
Really, ethernet in 0xe000 and wan in 0x2e?
So, 0xe006 and 0x28 are empty?

Reply:
MAC address setup is also same as WeVO W2914NS v2.
0x4: WiFi 2.4GHz (label_mac-8)
0x2e: WAN (label_mac)
0x8004: WiFi 5GHz (label_mac-4)
0xe000: LAN (label_mac+1)

Reply:
Ah, I remember. Please add your findings (might just copy your comment as-is) to the commit message.

Reply:
Okay, added.